src/astro/sql/operators/load_file.py

```diff
@@ -0,0 +1,179 @@
+from typing import Any, Dict, Optional, Union
+
+import pandas as pd
+from airflow.hooks.base import BaseHook
+from airflow.models import BaseOperator
+from airflow.models.xcom_arg import XComArg
+
+from astro.constants import DEFAULT_CHUNK_SIZE, LoadExistStrategy
+from astro.databases import BaseDatabase, create_database
+from astro.files import File, get_files
+from astro.sql.table import Table
+from astro.utils.task_id_helper import get_task_id
+
+
+class LoadFile(BaseOperator):
+    """Load S3/local table to postgres/snowflake database
+
+    :param input_file: File path and conn_id for object stores
+    :param output_table: Table to create
+    :param ndjson_normalize_sep: separator used to normalize nested ndjson.
+    :param chunk_size: Specify the number of records in each batch to be written at a time.
+    :param if_exists: Overwrite file if exists. Default False.
+
+    :return: If ``output_table`` is passed this operator returns a Table object. If not
+        passed, returns a dataframe.
+    """
+
+    template_fields = ("output_table", "input_file")
+
+    def __init__(
+        self,
+        input_file: File,
+        output_table: Optional[Table] = None,
+        chunk_size: int = DEFAULT_CHUNK_SIZE,
+        if_exists: LoadExistStrategy = "replace",
+        ndjson_normalize_sep: str = "_",
```

Comment:
We need to think about how we expose `ndjson_normalize_sep`. It is too specific to one `file_type`.